### PR TITLE
Add Invitation to Participate in Introduction

### PR DESCRIPTION
**Pull request by boyleconnor**

### CODE REVIEW

File: wiki/604825-homepage.md [hunk 1]
```diff
@@ -105,3 +105,5 @@ Join the [NLP Sandbox Discord Server] to chat and participate to live discussion
 [Become a partner organization]: https://www.synapse.org/#!Synapse:syn22277124/wiki/608958
 [NLP Sandbox PHI Deidentifier]: https://phi-deidentifier.nlpsandbox.io/
 [nlpsandbox/phi-deidentifier-app]: https://github.com/nlpsandbox/phi-deidentifier-app
+[participate by developing and submitting an NLP Sandbox tool]: https://www.synapse.org/#!Synapse:syn22277123/wiki/609136
```
Comment (tschaffter) [inline wiki/604825-homepage.md]:
```suggestion
[Participate by developing and submitting an NLP Sandbox tool]: https://www.synapse.org/#!Synapse:syn22277123/wiki/609136
```

File: wiki/604825-homepage.md [hunk 2]
```diff
@@ -105,3 +105,5 @@ Join the [NLP Sandbox Discord Server] to chat and participate to live discussion
...
+[browse the available tasks]: https://www.synapse.org/#!Synapse:syn22277123/wiki/609133
```
Comment (tschaffter) [inline wiki/604825-homepage.md]:
```suggestion
[Browse the available tasks]: https://www.synapse.org/#!Synapse:syn22277123/wiki/609133
```

**Review by tschaffter — APPROVED** [review]
Looks good to me! Just capitalized the first letter of new items

**Comment by boyleconnor** [comment]
NLP Sandbox is an open platform for benchmarking modular natural language
processing (NLP) tools on both public and private datasets. Academics,
students, and industry professionals are invited to [browse available
tasks] and participate by developing and submitting an NLP Sandbox tool
<https://www.synapse.org/#!Synapse:syn22277123/wiki/609136>.
It looks like there was a mistake and [browse available tasks] didn't get
turned into a proper link

**Comment by tschaffter** [comment]
@cascadianblue `browse the available tasks` vs `Browse available tasks`. Will submit a fix shortly.

**Pull request merged**